app/models/group.rb

```diff
@@ -0,0 +1,6 @@
+class Group < ApplicationRecord
+  has_many :entities, dependent: :destroy
```

Comment:
- [ ] 🤔 Since the user can have many Groups Why don't you add `belongs_to :user` into the Group Model?

Reply:
Solved thank you